src/AudacityLogger.h

```diff
@@ -18,47 +18,48 @@
 
 #include "Experimental.h"
 
-#include "MemoryX.h"
-#include "Prefs.h"
+#include <functional>
 #include <wx/log.h> // to inherit
 #include <wx/event.h> // to inherit wxEvtHandler
 
 class wxFrame;
 class wxTextCtrl;
 
 class AudacityLogger final : public wxEvtHandler,
-                             public wxLog,
-                             public PrefsListener
+                             public wxLog
 {
  public:
  
    // Get the singleton instance or null
    static AudacityLogger *Get();
 
-   void Show(bool show = true);
-
    bool SaveLog(const wxString &fileName) const;
    bool ClearLog();
 
+   //! Retrieve all or some of the lines since most recent ClearLog or start of program
+   /*! If `count == 0` or is more than the number of lines, return all; else return the last `count` lines */
    wxString GetLog(int count = 0);
```

Comment:
May be rename `count` to more clearly, something like: `linesNumb`?

Reply:
Perhaps size_t numLines -- also using an unsigned type not int.